pkg/user/repository/RoleGroupRepository.go

```diff
@@ -123,6 +123,11 @@ func (impl RoleGroupRepositoryImpl) GetRoleGroupListByCasbinNames(names []string
 	err := impl.dbConnection.Model(&model).Where("casbin_name in (?)", pg.In(names)).Where("active = ?", true).Select()
 	return model, err
 }
+func (impl RoleGroupRepositoryImpl) GetRoleGroupByCasbinName(name string) (*RoleGroup, error) {
+	var model RoleGroup
+	err := impl.dbConnection.Model(&model).Where("casbin_name = ?", name).Where("active = ?", true).Order("updated_on desc").Select()
```

Comment:
what is the need of ordering? if there can be multiple rows in output use limit with this or use slice model